karma.conf.js

```diff
@@ -53,7 +53,16 @@ module.exports = function(config) {
 
       "website/public/js/filters/filters.js",
 
-      "website/public/js/directives/directives.js",
```

Comment:
Is there a reason to not do `"website/public/js/directives/*.js"` or `"website/public/js/directives/**/*.js"`? Other than the obvious, that we're not doing this anywhere else other than in the spec files.